.github/workflows/dev-release.yml

```diff
@@ -0,0 +1,48 @@
+name: Release @dev tag to npm
+on:
+  pull_request:
+    types: [labeled]
+
+jobs:
+  release:
+    if: ${{ contains(github.event.pull_request.labels.*.name, 'release dev tag') }}
```

Comment:
maybe you can check with label was ADDED in this run (payload) instead what are all labels?

so 
1. "foobar" added → do nothing
2. "release dev tag" added → release
3. "barbaz" added → **do nothing, but currently it will release again**

Reply:
Good idea fixed - now we check the last label added in event. It means if you already have `release dev tag` label and add other label workflow will not be triggered.